demos/ASP.Net MVC/Divergent.Frontend/Views/Orders/Details.cshtml

```diff
@@ -16,12 +15,3 @@
         <div>Courier: @Model.ShippingCourier</div>
     </div>
 </div>
-
-@*<div class="panel panel-default">
-    <div class="panel-heading">
-        <h3 class="panel-title">Shipment information</h3>
-    </div>
-    <div class="panel-body">
-
-    </div>
-</div>*@
```

Comment:
There are many ways to do it. Personally I'd open the TortoiseGit log view, then diff the tip of the branch to master, and then right click each file and "Revert to 2cbcea1".